rules/REQUEST-934-APPLICATION-ATTACK-GENERIC.conf

```diff
@@ -354,6 +354,39 @@ SecRule REQUEST_COOKIES|!REQUEST_COOKIES:/__utm/|REQUEST_COOKIES_NAMES|ARGS_NAME
     setvar:'tx.inbound_anomaly_score_pl2=+%{tx.critical_anomaly_score}'"
 
 
+# [ Python generic RCE signatures ]
+#
+# Detects Python SSTI attacks.
+# Example: {{['cat\x20/etc/passwd']|filter('system')}}
+#
+# Regular expression generated from regex-assembly/TODO
```

Comment:
This part until line 365 can be removed, as the regex is not generated. It's too short.